src/python/analyzer_executor/src/analyzer_executor_lib/BUILD

```diff
@@ -1 +1,5 @@
-python_library()
+python_library(
+    dependencies = [
+        "src/python/python-proto",
```

Comment:
Can you play with it and see if this needs to be explicit? 

I think it's "broken-ish" if you use `from graplinc` but it should be able to pick up `from python_proto` implicitly just fine